src/archiefvernietigingscomponent/report/utils.py

```diff
@@ -89,7 +94,7 @@ def get_process_owner_comments(destruction_list: DestructionList) -> str:
     return review.text
 
 
-def create_destruction_report(destruction_list: DestructionList) -> str:
+def create_destruction_report_content(destruction_list: DestructionList) -> str:
     destroyed_items = destruction_list.items.filter(
```

Comment:
I think the content of the document should look more official: it should include the title, the metadata of DL (name, date of processing) etc
Perhaps the organization name, its logo and contact details could also be used 

The report formatting can be handled in the next PR, since it could be tricky to do for PDF

Reply:
In the PR where I turned the report into PDF, I made it look like the screen shot in the ticket (#55), except with more columns. Maybe we should clarify with the product owners what should be the style of the document (since I think it is only an attachment to another document?)